std_msgs/Header header
FullDetection[] detections